inc/item_remotemanagement.class.php

```diff
@@ -0,0 +1,370 @@
+         $header .= "</th>";
+         $header .= "<th>".__('Remote ID')."</th>";
+         $header .= "<th>".__('Type')."</th>";
+         $header .= "<th>".__('Automatic inventory')."</th>";
+         $header .= "</tr>";
+         echo $header;
+
+         Session::initNavigateListItems(
+            __CLASS__,
+            //TRANS : %1$s is the itemtype name,
+            //        %2$s is the name of the item (used for headings of a list)
+            sprintf(
+               __('%1$s = %2$s'),
+               $item::getTypeName(1),
+               $item->getName()
+            )
+         );
+
+         $mgmt = new self();
+         while ($data = $iterator->next()) {
+            $mgmt->getFromResultSet($data);
+            echo "<tr class='tab_bg_2'>";
+
+            echo "<td width='10'>";
+            Html::showMassiveActionCheckBox(__CLASS__, $data["id"]);
+            echo "</td>";
+            echo "<td>".$mgmt->getRemoteLink()."</td>";
+            echo "<td>".$mgmt->fields['type']."</td>";
+            echo "<td>".Dropdown::getYesNo($data['is_dynamic'])."</td>";
+            echo "</tr>";
+            Session::addToNavigateListItems(__CLASS__, $data['id']);
+         }
+         echo $header;
+      } else {
+         echo "<tr class='tab_bg_2'><th colspan='$colspan'>".__('No item found')."</th></tr>";
+      }
+
+      echo "</table>";
+      echo "</div>";
+   }
+
+
+   /**
+    * Get remote management system link
+    *
+    * @return string
+    */
+   public function getRemoteLink(): string {
+      $link= '<a href="%s" target="_blank">%s</a>';
+      $id = $this->fields['remoteid'];
+      $href = null;
+      switch ($this->fields['type']) {
+         case self::TEAMVIEWER:
+            $href = "https://start.teamviewer.com/$id";
+            break;
+         case self::ANYDESK:
+            $href = "anydesk:$id";
+            break;
+      }
+
+      if ($href === null) {
+         return $id;
+      } else {
+         return sprintf(
+            $link,
+            $href,
+            $id
+         );
+      }
+   }
+
+   function rawSearchOptions() {
+
+      $tab = [];
+
+      $tab[] = [
+         'id'                 => 'common',
+         'name'               => __('Characteristics')
+      ];
+
+      $tab[] = [
+         'id'                 => '1',
+         'table'              => $this->getTable(),
+         'field'              => 'remoteid',
+         'name'               => __('ID'),
+         'datatype'           => 'itemlink',
+         'massiveaction'      => false,
+         'autocomplete'       => true,
+      ];
+
+      $tab[] = [
+         'id'                 => '2',
+         'table'              => $this->getTable(),
+         'field'              => 'type',
+         'name'               => __('Type'),
+         'datatype'           => 'string',
+         'massiveaction'      => false,
+         'autocomplete'       => true,
+      ];
```

Comment:
`type` has limited values, SO should use a specific datatype, and should not have the `autocomplete` property.

Reply:
Values fot type are not limited, this is just a string in the specification.